.circleci/config.yml

```diff
@@ -27,3 +27,14 @@ jobs:
           key: dependency-cache-{{ checksum "package.json" }}
           paths:
             - node_modules
+workflows:
+  version: 2
+  commit-workflow:
+    jobs:
+      - build
+  scheduled-workflow:
+    triggers:
+      - schedule:
```

Comment:
Should we add a filter to run on a specific branch or is that part of the config of the job already ?

Reply:
@lcampos - Good question. It's for all branches now. The syntax allows us to limit it. See https://circleci.com/docs/2.0/workflows/#scheduling-a-workflow

I'll limit it to develop (and master since we might need to do e-releases from there).